src/components/SavingMessage.tsx

```diff
@@ -0,0 +1,17 @@
+import { ReactNode } from "react";
+import "./SavingMessage.css";
+
+interface SavingMessageProps {
+  children: string | ReactNode;
+}
+
+export const SavingMessage = ({ children }: SavingMessageProps) => {
+  return (
+    <div className="blob-wrapper">
+      <div className="blob white"></div>
+      <span className="saving">
+        <span>{children}</span>
+      </span>
```

Comment:
Is the double _span_ necessary? Maybe this is enough:

```suggestion
    <div className="blob-wrapper">
      <div className="blob white"></div>
      <span className="saving">
        {children}
      </span>
```

Also, I am not sure if _span_ is a good element to contain the children because children can be anything (a _div_, a _p_, or an _image_). What do you think about using a _div_ or simply nothing?
```suggestion
    <div className="blob-wrapper saving">
      <div className="blob white"></div>
      {children}
```

Reply:
Now is 👇🏻 

> 
    <div className="blob-wrapper">
      <div className="blob white"></div>
      <span className="saving">
        <span>
          <FormattedMessage id="saving" />
        </span>
      </span>
    </div>

Reply:
In that case, we will not receive arbitrary elements (like a _div_, a _p_, or an _image_), 👍. 

And what about the double _span_? is removing a spam possible?

```diff
  <div className="blob-wrapper">
    <div className="blob white"></div>
    <span className="saving">
-     <span>
        <FormattedMessage id="saving" />
-     </span>
    </span>
  </div>
```

Reply:
done